activerecord/lib/active_record/statement_cache.rb

```diff
@@ -74,13 +74,13 @@ def <<(str)
         self
       end
 
-      def add_bind(obj)
+      def add_bind(obj, &_)
         @binds << obj
         @parts << Substitute.new
         self
       end
 
-      def add_binds(binds, proc_for_binds = nil)
+      def add_binds(binds, proc_for_binds = nil, &_)
```

Comment:
We require 3.2 now, so you can use an anonymous block

```suggestion
      def add_binds(binds, proc_for_binds = nil, &)
```